### PR TITLE
add no build option

### PR DESCRIPTION
**Pull request by shibuiwilliam**
# What
- Add --no-build option to record session which is added from Launchable CLI v1.59.0.
- Ref. https://github.com/launchableinc/cli/releases/tag/v1.59.0

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -68,8 +73,17 @@ runs:
     - id: record_test
       shell: bash
       run: |
-        launchable record tests \
-          --build ${{ inputs.build_name }} \
-          --allow-test-before-build \
-          ${{ inputs.test_runner }} \
-          ${{ inputs.report_path }}
+        if [ "${{ inputs.no_build }}" == "false" ]; then
```
Comment (Konboi) [inline action.yaml:76]: nits: Wouldn't it be easier to understand it the other way around?

```
if [ "${{ inputs.no_build }}" == "true" ]; then
  launchable record tests --no-build
else 
  launchable record tests 
fi
```

Comment (shibuiwilliam) [reply action.yaml]: fixed
https://github.com/launchableinc/record-build-and-test-results-action/pull/5/commits/7b233720d29a3c764e9cfa9d040e7befe338bf4a

File: .github/workflows/go-test-example.yaml [hunk 2]
```diff
@@ -28,10 +28,23 @@ jobs:
...
           test_runner: go-test
           source_path: ./examples/go/
         if: always()
+      - name: remove .launchable
+        run: rm -f .launchable
+      - name: Record test results without build
+        uses: ./
+        # uses: launchableinc/record-build-and-test-results-action@v1.0.0
+        with:
+          build_name: $GITHUB_RUN_ID
```
[the remark this answers is not in the document]
Comment (shibuiwilliam) [reply .github/workflows/go-test-example.yaml]: fixed
https://github.com/launchableinc/record-build-and-test-results-action/pull/5/commits/7b233720d29a3c764e9cfa9d040e7befe338bf4a

File: action.yaml [hunk 3]
```diff
@@ -68,8 +73,17 @@ runs:
...
+          launchable record tests \
+            --build ${{ inputs.build_name }} \
+            --allow-test-before-build \
+            ${{ inputs.test_runner }} \
+            ${{ inputs.report_path }}
+        else
+          launchable record tests \
+            --build ${{ inputs.build_name }} \
```
[the remark this answers is not in the document]
Comment (shibuiwilliam) [reply action.yaml:84]: fixed
https://github.com/launchableinc/record-build-and-test-results-action/pull/5/commits/7b233720d29a3c764e9cfa9d040e7befe338bf4a

**Review by Konboi — APPROVED** [review]

**Pull request merged**